src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import duke.commands.Command;
+
+/**
+ * The main class of this chatting bot programme.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+
+    /**
+     * The constructor of this class.
+     */
+    public Duke() {
+        storage = new Storage();
+        tasks = storage.load();
+    }
+
+
+    /**
+     * You should have your own function to generate a response to user input.
+     * Replace this stub with your completed method.
+     */
```

Comment:
Maybe you could replace this placeholder comment with information more specific to the function the way you've implemented it?